inst/settings/resultsDataModelSpecification.csv

```diff
@@ -121,6 +121,7 @@ index_event_breakdown,database_id,varchar,Yes,Yes,No,Yes
 index_event_breakdown,domain_field,varchar,Yes,Yes,No,Yes
 index_event_breakdown,domain_table,varchar,Yes,Yes,No,Yes
 metadata,database_id,varchar,Yes,Yes,No,Yes
+metadata,start_time,varchar,Yes,Yes,No,Yes
```

Comment:
does isRequired need to be set to true?

Reply:
metadata table changes wont break DiagnosticsExplorer because it is not used by DiagnosticsExplorer. Metadata file was mostly explored as a csv file

Reply:
You are right, this maybe an optional field